Services/Membership/classes/class.ilParticipant.php

```diff
@@ -307,14 +316,12 @@ protected function readParticipant()
         $users = array();
         $this->participants = array();
         $this->members = $this->admins = $this->tutors = array();
-        
-        $member_roles = array();
 
```

Comment:
Please can you add $this->member_roles = [];
If, for any reason this method is called twice, the number of member roles would be duplicated without this line.